readthedocs/restapi/templates/restapi/log.txt

```diff
@@ -0,0 +1,13 @@
+RTD build information
+Build: {{ build.id }}
+Project: {{ build.project }}
+Version: {{ build.commit }}
+Date: {{ build.date }}
```

Comment:
Also, is the default format for dates fine?

Reply:
Yes! :)

Reply:
Maybe this is better, to keep compatibility with other places: https://github.com/rtfd/readthedocs.org/blob/382c593f2630d235b020434e63d5f543a9059c10/readthedocs/templates/builds/build_detail.html#L38